internal/errors.go

```diff
@@ -44,15 +44,27 @@ func TranslateError(err error, isCloudCluster bool, op ...string) (string, bool)
 }
 
 func TranslateClusterError(err error, operation string) (string, bool) {
-	var urlError *url.Error
-	if errors.As(err, &urlError) && strings.Contains(urlError.Error(), "EOF") {
+	var urlErr *url.Error
+	restAPIErr := `Cannot access Hazelcast REST API.
```

Comment:
How about `errMsg` instead of `restAPIErr`?

Reply:
I would move the error message contents (`errMsg`) outside of the function to a constant.

Reply:
Moved both messages to a const block: https://github.com/hazelcast/hazelcast-commandline-client/pull/35/commits/b8ea8c4e30cfbde45e4529f4b2926ae3600dee17